imsv-docs-astro/src/content/docs/guides/testing/passing-kyc-in-test.mdoc

```diff
@@ -0,0 +1,77 @@
+---
+title: Passing KYC in Test
+description: Instructions for how to navigate KYC in test to obtain a pass on the Immersve platform for each KYC mode.
+slug: guides/pass-kyc-in-testmode
+---
+
+This guide explains how to provide KYC data to our test environment to consistently pass KYC. This is useful
+for getting onto the platform and testing your integrations.
+
+## Partner Conducted KYC
+
+When submitting a partner statement as part of {% link page="guides/partner-conducted-kyc" /%},
+to consistently pass KYC in test mode, you will need to supply a unique identity and unique address.
+To achieve this we will be replacing some parts of the partner statement with random values
+and key words within different claims.
+
+{% note %}
+ Leverage random functions or libraries in your respective programming languages
+ to help generate these values.
+ e.g. `fakerjs` in javascript, random functions like `$randomFirstName`, `$randomLastName` in postman etc.
+ The values do not need to be real, just random and valid for their respective types.
+{% /note %}
+
+ {% endpointref name="submit-kyc-statement" /%}
+
+### Adjust Full Name claim
+Generate random values for the following fields:
+ - `givenName`
+ - `familyName`
+
+ Ensure the `middleName` is `passall`.
+### Adjust Date of Birth claim
+Generate a random date in the past for the `dateOfBirth` field
+### Adjust Address claim
+We recommend the following fields have random values:
+  - `streetNumber`
+  - `streetName`
+  - `town`
+
+Make sure that the other geographic details (`region`, `state`, `country`) of the
+address align with the kyc region you are submitting the statement for.
+
+## Immersve Conducted KYC
+
+To pass KYC in test. You should not need to do anything different from the normal process.
+A test document is available for use instead of your own identity documents, however
+you will still need to use a real person's face for the facial scan.
+
+For a complete guide on passing KYC in test mode, please read {% link page="guides/immersve-conducted-kyc" /%}.
```

Comment:
I don't think this makes sense @forktrucka ? This page is the complete guide to passing KYC in test mode, should this say "For more a complete guide on Immersve Conducted KYC please read <link>?"

Reply:
yeah sorry, that was the text from your earlier feedback

Reply:
Would a generic `See {% link page="guides/immersve-conducted-kyc" /%} for more info.` be better?

Reply:
Sounds good, though I'd use 'information' over the abbreviated version.

Reply:
this has once again changed.